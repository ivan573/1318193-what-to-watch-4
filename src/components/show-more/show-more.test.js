import React from "react";
import renderer from "react-test-renderer";
import ShowMore from "./show-more.jsx";

describe(`Show more button matches snapshot`, () => {
  it(`Show more button renders if not all movies are displayed`, () => {
    const tree = renderer
    .create(<ShowMore
      areAllMoviesShown={false}
      onShowMoreClick={() => {}}
    />).toJSON;

    expect(tree).toMatchSnapshot();
  });

  it(`Show more button does not render if all movies are displayed`, () => {
    const tree = renderer
    .create(<ShowMore
      areAllMoviesShown={true}
      onShowMoreClick={() => {}}
    />).toJSON;

    expect(tree).toMatchSnapshot();
  });

});
