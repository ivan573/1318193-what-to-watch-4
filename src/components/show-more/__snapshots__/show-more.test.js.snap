// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show more button matches snapshot Show more button does not render if all movies are displayed 1`] = `[Function]`;

exports[`Show more button matches snapshot Show more button renders if not all movies are displayed 1`] = `[Function]`;
